app/grants/urls.py

```diff
@@ -19,13 +19,14 @@
 """
 from django.urls import path, re_path
 
-from grants.views import grant_details, grant_fund, grant_new, grants, subscription_cancel
+from grants.views import grant_details, grant_fund, grant_new, grants, milestones, subscription_cancel
 
 app_name = 'grants'
 urlpatterns = [
     path('', grants, name='grants'),
     path('<int:grant_id>', grant_details, name='details'),
     re_path(r'^new', grant_new, name='new'),
+    path('milestones/<int:grant_id>', milestones, name='milestones'),
```

Comment:
Can we go with `<int:grant_id>/milestones` for this?

Reply:
That was the [original approach](https://github.com/gitcoinco/web/pull/2395/commits/1d3b1fedef6960fc6dcc6ca515d3955f2ec699ac#diff-f821398ceeaa8b31a0d99c3bb285efe6), but I changed it base on [@captnseagraves's comment](https://github.com/gitcoinco/web/pull/2395#issuecomment-428640351):
> In grants/urls.py could you change ln32 to have the same pattern as ln33 to maintain consistency?

I'm fine with either approach, just let me know what works you both.